docker/Dockerfile

```diff
@@ -22,8 +24,8 @@ RUN apt-get install -y wget curl git python3 make gcc build-essential
 
 # Install Golang
 RUN apt-get install -y bison
-RUN wget https://go.dev/dl/go1.20.10.linux-${ARCH}.tar.gz
-RUN rm -rf /usr/local/go && tar -C /usr/local -xzf go1.20.10.linux-${ARCH}.tar.gz
+RUN wget https://go.dev/dl/go${GO_VERSION}.linux-${ARCH}.tar.gz
```

Comment:
If I'm understanding `setGO_VERSION` and `getGO_VERSION` correctly, we just parse the line in `go.mod` containing the Go version. However, that line only contains the major and minor versions (not the bug fix version), so I'm concerned these links may not work. For instance `wget https://go.dev/dl/go1.21.darwin-amd64.tar.gz` returns 404.

Reply:
The first part, `$(getDepVersion go)` grabs the major/minor, and then `grep -m1 GO_PATCH_VERSION $TELEPORTER_PATH/go.mod | cut -d ' ' -f3` grabs the patch version I've left in the comment in `go.mod`.

Reply:
Nice 👍